src/api/app/models/package.rb

```diff
@@ -968,7 +968,7 @@ def valid_name
     errors.add(:name, 'is illegal') unless Package.valid_name?(self.name)
   end
 
-  def branch_from(origin_project, origin_package, rev = nil, missingok = nil, comment = nil)
+  def branch_from(origin_project, origin_package, rev = nil, missingok = nil, comment = nil, linkrev = nil)
```

Comment:
That might be a good point to replace some of the parameters by an options hash:)